tests/sentry/relay/test_config.py

```diff
@@ -29,26 +29,28 @@
 
 @pytest.mark.django_db
 @pytest.mark.parametrize("full", [False, True])
-def test_get_project_config(default_project, insta_snapshot, full):
+@pytest.mark.parametrize("has_ops_breakdown", [False, True])
```

Comment:
There are likely unused snapshot files in ` tests/sentry/relay/snapshots/test_config/test_get_project_config/` because of this change. Also I would encourage to:

```suggestion
@pytest.mark.parametrize("has_ops_breakdown", [False, True], ids=["ops_breakdown", "no_ops_breakdown"])
```

for better filenames. likewise with `full` param